zanata-war/src/main/java/org/zanata/security/ZanataOpenId.java

```diff
@@ -189,10 +189,19 @@ public String verifyResponse(HttpServletRequest httpReq) {
             // clear any previous messages which might have been generated
             // before the openId authentication took place
             facesMessages.clear();
-            // extract the parameters from the authentication response
-            // (which comes in as a HTTP request from the OpenID providerType)
+            /**
+             * extract the parameters from the authentication response query string
+             * (which comes in as a HTTP request from the OpenID providerType)
+             * instead of from httpReq.getParameterMap() to make sure params
+             * is encoded correctly bypassing default servlets encoding.
+             *
+             * httpReq.getParameterMap() failed check in
+             * {@link org.openid4java.consumer.ConsumerManager#verifySignature}
+             * due to the unicode encoding in URI is different from signature.
+             * Reported issue: https://zanata.atlassian.net/browse/ZNTA-1275
```

Comment:
When I said to put the "reported issue" into the comment, I meant the issue which someone reported against openid4java (or forum link). The link we were looking at yesterday. 

@aeng Could you please find the link we were looking at yesterday, and link to it from the Zanata issue?